doc/source/io.rst

```diff
@@ -2228,9 +2228,10 @@ round-trippable manner.
    new_df
    new_df.dtypes
 
-Please note that the string `index` is not supported with the round trip
-format, as it is used by default in ``write_json`` to indicate a missing index
-name.
+Please note that the literal string 'index' as the name of an ``Index`` is not
+round-trippable, nor are any names beginning with 'level_' within a
+``MultiIndex``. These are used by default in ``write_json`` to indicate missing
```

Comment:
can you use ``:class`Index` `` and ``:func:`DataFrame.to_json` `` (you mean that instead of write_json I think)